runtime/drivers/registry.go

```diff
@@ -136,6 +140,8 @@ func (i *Instance) Config() (InstanceConfig, error) {
 		ModelDefaultMaterialize:           false,
 		ModelMaterializeDelaySeconds:      0,
 		MetricsApproximateComparisons:     true,
+		MetricsApproximateComparisonsCTE:  false,
+		MetricsTwoPhaseComparisons:        true,
```

Comment:
Would it make sense to differentiate between queries with a small limit and (unlimited) queries like exports? Since I guess the two-phase approach will not be suitable for high cardinality results?

If yes, do you think it would be best to keep falling back to the CTE approach for unlimited queries, or should it just fall back to the non-CTE approach?

Reply:
I think it should fall back to non-CTE approach which works faster than CTE in general. However it causes issues with druid dimensions having high cardinality column so for those customer we can enable CTE approach. Best would to be have non-CTE enabled, monitor for ResourceLimitExceeced exception and automatically fall back to CTE approach